docs/source/tutorial.rst

```diff
@@ -0,0 +1,3 @@
+Tutorial
+========
+First, clone our github repository with 'git clone https://github.com/MSDS698/2020-product-analytics-group-project-sparkle.git', then run 'cd 2020-product-analytics-group-project-sparkle && sh deploy.sh' to run our web app on Elastic Beanstalk!
```

Comment:
Sadly the classroom repo is private and not available for cloning 😣. We'll have to update this tutorial to work with the public repo later